.github/workflows/on-workflow-call-test.yml

```
@@ -27,52 +74,84 @@ jobs:
+            fi
+            echo "Waiting for Neon proxy ($i)..."
+            sleep 2
+          done
+
       - uses: actions/checkout@v4
+
       - uses: ./.github/actions/pnpm-install
+        with:
+          node-version: 20 # or matrix like above
+
       - uses: ./.github/actions/setup-env
```

actionlint (1.7.7)

missing input "NEXT_PUBLIC_BLOG_URL" which is required by action "Setup Environment Variables" defined at "./.github/actions/setup-env". all required inputs are "DATABASE_URL", "KIT_API_KEY", "NEXT_PUBLIC_BLOG_GOOGLE_ANALYTICS_ID", "NEXT_PUBLIC_BLOG_URL", "NEXT_PUBLIC_POSTHOG_HOST", "NEXT_PUBLIC_POSTHOG_KEY", "NEXT_PUBLIC_SENTRY_DSN", "NEXT_PUBLIC_WWW_GOOGLE_ANALYTICS_ID", "NEXT_PUBLIC_WWW_URL", "NODE_ENV", "S3_BUCKET_ACCESS_KEY_ID", "S3_BUCKET_NAME", "S3_BUCKET_REGION", "S3_BUCKET_SECRET_KEY", "S3_BUCKET_URL", "SENTRY_AUTH_TOKEN", "SENTRY_ORG", "SENTRY_PROJECT"

(action)